Connection.php

```diff
@@ -647,8 +653,20 @@ public function executeCommand($name, $params = [])
         }
 
         \Yii::trace("Executing Redis Command: {$name}", __METHOD__);
+        if ($this->retries > 0) {
+            $retries = $this->retries;
+            while ($retries-- >= 0) {
+                try {
+                    fwrite($this->_socket, $command);
+                    return $this->parseResponse(implode(' ', $params));
```

Comment:
Good morning myself :( I was looking at `$retries--` seeing `$this->retries--`. Sorry.